### ISSUE TITLE
Priest: Shadowfiend Damage

### ISSUE THREAD

**Issue by Himalaye** [dalaranwow/dalaran-wow#4939]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1.  Priest issue: Shadowfiend ist doing close to no dmg.
2.  This happens since you added some changes again on Friday the 20th of October.
3.  Please Fix ASAP its a really imporant CD for the Priest especially Shadow.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by AttendantTock** [comment]
Hello,

Indeed, the shadowfiend doesn't seem to care about caster's stats anymore..

Full BiS:
<img width="1785" alt="capture d ecran 2017-10-22 a 22 09 09" src="https://user-images.githubusercontent.com/29557022/31866297-a8cc9002-b775-11e7-8402-fa5818f225fb.png">

Naked:
<img width="1728" alt="capture d ecran 2017-10-22 a 22 12 48" src="https://user-images.githubusercontent.com/29557022/31866331-2cc9fb42-b776-11e7-9c92-15e4f4ac8632.png">

**Comment by Himalaye** [comment]
oh, sorry I thought it should scale with it..
since it did since the beginning of dala so I thought it was normal

**Comment by Lothloryen** [comment]
Hello, thank you @AttendantTock for looking into it. @Himalaye No, no, what he means is that you are right, we will check it out.

**Comment by Himalaye** [comment]
oh nice
I thought I missunderstood it so I deleted my comment right after x)
thanks loth!

**Comment by AttendantTock** [comment]
Yes, it's supposed to scale with it:

http://www.wowhead.com/spell=34433/shadowfiend#comments
> Ulduar 3.1 Shadowfiend
> Shadowfiend's dps scales with your spell power now (receives 30% of caster's spell power in damage). New Shadowfiend ability, Shadowcrawl, teleports the Shadowfiend to an enemy target and increases its damage done by 15% for 5 seconds.

**Comment by robinsch** [comment]
Woops, I'm sorry had the wrong auras IDs.

**Comment by TeridanDW** [comment]
Works as intended on PTR.

**Comment by Lothloryen** [comment]
It's already on live, thanks everyone.
